custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE "{new_table}" B
+SET
+pse_daily_attendance = A.pse_days_attended,
+pse_daily_attendance_male = CASE WHEN sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+pse_daily_attendance_female = CASE WHEN sex = 'F' THEN A.pse_days_attended ELSE NULL END
+FROM "{old_table}" A
+WHERE A.doc_id = B.doc_id and A.month = immutable_date_cast(B.month_start)
```

Comment:
we actually added an expression index on (doc_id, immutable_date_cast(month_start)), because the relevant columns in each table were of different types. so there is an index on that, although explain did not appear to use it. perhaps because the column being written isnt in the index. I tested an EXPLAIN on a join on just doc_id and it also did not plan on using the index.